packages/api/src/services/utils/routing-utils.ts

```diff
@@ -40,12 +40,24 @@ export function isHomePage(): boolean {
 }
 
 /**
- * Retrieves the pathname portion of the current URL.
+ * Retrieves the pathname portion of the current URL without the context prefix.
  *
- * @returns {string} The pathname of the current URL, which represents the path segment that comes after the host and before the query string.
+ * @returns {string} The pathname of the current URL, which represents the path segment that comes after the context (if any) and before the query string.
  */
 export function getPathName(): string {
-  return window.location.pathname;
+  return window.location.pathname.substring(getContextName().length - 1);
```

Comment:
Does this also fix GDB-12612?

Reply:
OK cool. @plamen-yordanov you should close the other MR.

Also please change the name of the MR to better reflect what is done, as this probably fixes other problems and not just the request.